core/script/rails

```diff
@@ -1,8 +0,0 @@
-#!/usr/bin/env ruby
-# This command will automatically be run when you run "rails" with Rails 3 gems installed from the root of your application.
-
-ENGINE_ROOT = File.expand_path('../..', __FILE__)
-ENGINE_PATH = File.expand_path('../../lib/spree/core/engine', __FILE__)
-
-require 'rails/all'
-require 'rails/engine/commands'
```

Comment:
We should probably cherry-pick this out into another PR.

Reply:
Just pulled this out entirely, can be dealt with later